installers/windows.gradle

```diff
@@ -48,11 +48,10 @@ task agentWindowsExe(type: WindowsPackagingTask, dependsOn: [':agent-bootstrappe
     project.copy { from('go-agent/release/start-agent.bat') into(versionedDir()) }
     project.copy { from('go-agent/release/stop-agent.bat') into(versionedDir()) }
     project.copy { from('go-agent/release/README.md') into(versionedDir()) }
-    project.copy { from('go-agent/win/cruisewrapper.exe') into(versionedDir()) }
+    project.copy { from("windows-shared/tanuki/${flavour()}/cruisewrapper.exe") into(versionedDir()) }
```

Comment:
Building a 64 bit installer meant taking care of migration of existing installation in an automatic fashion during upgrades/server startup or by providing some script to do the same or at the very least by giving a heads up to the existing users. Not in favour of doing it automatically as the server installation folder contains flyweights and artifacts which could be huge in size and might take forever to migrate. We should give an advance notice to the users regarding the same, but it did not make sense to stop this piece of work while we figure out a migration strategy. As of now, the 64bit installers package 64bit jre and 32bit one packages the 32 bit jre. 
As far as naming the installers is concerned, since the installers are not really 64 bit, it did not make sense to call them x64 or x86.